crates/sui-indexer/src/framework/runner.rs

```diff
@@ -5,18 +5,24 @@ use sui_rest_api::CheckpointData;
 
 use super::interface::Handler;
 
-pub async fn run<S>(mut stream: S, mut handlers: Vec<Box<dyn Handler>>)
+pub async fn run<S>(stream: S, mut handlers: Vec<Box<dyn Handler>>)
 where
     S: futures::Stream<Item = CheckpointData> + std::marker::Unpin,
 {
     use futures::StreamExt;
 
-    while let Some(checkpoint) = stream.next().await {
+    let batch_size = std::env::var("CHECKPOINT_PROCESSING_BATCH_SIZE")
+        .unwrap_or(25.to_string())
```

Comment:
curious did you play around the numbers and picked 25 here?

Reply:
it's really a random number, the best number changes along with the checkpoint size